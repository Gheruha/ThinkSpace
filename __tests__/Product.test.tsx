import { render } from '@testing-library/react';
import Product from '@/app/product/page';

it('should load Product page', () => {
	const { container } = render(<Product />);
	expect(container).toMatchSnapshot();
});
