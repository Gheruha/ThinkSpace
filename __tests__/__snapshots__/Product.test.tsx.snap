// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load Product page 1`] = `
<div>
  <div
    class="hero-page"
  >
    <h1
      class="hero-text"
    >
      Organize your 
      <span
        class="text-violet-700"
      >
        Life
      </span>
    </h1>
    <p
      class="text-gray-800 dark:text-gray-200 max-w-[40rem] simple-text"
    >
      All in one single place, use all the templates available or create one by yourself.
    </p>
    <button
      class="items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-primary text-primary-foreground hover:bg-primary/90 h-10 px-4 py-2 flex space-x-1 get-started"
    >
      <span>
        Get Started
      </span>
      <svg
        class="lucide lucide-arrow-right"
        fill="none"
        height="20"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="3"
        viewBox="0 0 24 24"
        width="20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5 12h14"
        />
        <path
          d="m12 5 7 7-7 7"
        />
      </svg>
    </button>
  </div>
</div>
`;
